service/web/server/docs/openapi.yaml

```diff
@@ -0,0 +1,623 @@
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: "Metadata for measurements retrieved successfully."
+                  payload:
+                    $ref: '#/components/schemas/BoundedSignalMetaResponse'
+                  status:
+                    type: integer
+                    example: 200
+  /forecast_points/{testid}:
+    parameters:
+      - $ref: '#/components/parameters/TestId'
+    get:
+      summary: Get metadata for forecast points
+      tags:
+        - meta
+      description: Returns metadata for forecast point signals.
+      responses:
+        '200':
+          description: Successful response with metadata for forecast points.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: "Metadata for forecast points retrieved successfully."
+                  payload:
+                    $ref: '#/components/schemas/SignalMetaResponse'
+                  status:
+                    type: integer
+                    example: 200
+  /scenario/{testid}:
+    parameters:
+      - $ref: '#/components/parameters/TestId'
+    get:
+      summary: Get the current scenario description
+      tags:
+        - meta
+      description: Returns a description of the current scenario.
+      responses:
+        '200':
+          description: Successful response with the scenario description.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: "Scenario description retrieved successfully."
+                  payload:
+                    type: object
+                    additionalProperties:
+                      type: string
+                      description: Key-value pairs describing the current scenario.
+                  status:
+                    type: integer
+                    example: 200
+    put:
+      summary: Set the test scenario
+      tags:
+        - meta
+      description: Set the scenario for the running test. Optional properties may trigger an initialization (see README).
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                electricity_price:
+                  type: string
+                  description: Identifier for electricity price scenario.
+                time_period:
+                  type: string
+                  description: Named time period for scenario (may perform initialization).
+                temperature_uncertainty:
+                  type: string
+                  description: Temperature uncertainty setting.
```

Comment:
Suggest consistency in description as with all others, and more detail.  Use "Identifier for outside dry bulb temperature forecast uncertainty scenario."